scalafix-tests/input/src/main/scala-2/test/explicitResultTypes/ExplicitResultTypesOnlyImplicits.scala

```diff
@@ -1,6 +1,7 @@
 /*
 rules = ExplicitResultTypes
 ExplicitResultTypes.onlyImplicits = true
+ExplicitResultTypes.skipSimpleDefinitions = []
```

Comment:
we're not testing `skipSimpleDefinitions` in this test, hence overwriting the default